### PR TITLE
Fix #5, disallow a variable declaration without initializer

### LINKED ISSUE #5: SegmentationFault when variable declaration without inital value.
# Environment

```console
$ dmd --version
DMD64 D Compiler v2.090.0
Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved written by Walter Bright
$ uname -msrv
Linux 4.15.0-72-generic #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 x86_64
```

# How to reproduce

```console
$ git rev-parse HEAD
92eac556a7a9009638ba8aadc800c3ba03b855d0
$ ./bin/repl-d
> int a;
zsh: segmentation fault (core dumped)  ./bin/repl-d
```

### PR DESCRIPTION
**Pull request by kubo39**

### CODE REVIEW

**Comment by kubo39** [comment]
tarvis ci is unstable :/

File: source/repld/runner.d [hunk 1]
```diff
@@ -66,6 +66,9 @@ class REPLRunner {
         try {
             if (auto decl = parseResult.peek!(VariableDeclaration)) {
                 foreach (d; decl.declarators) {
+                    if (d.initializer is null) {
+                        return fail("Currently a variable declaration requires initializer.");
+                    }
                     evaluator.evalVarDecl(getType(*decl), d.name.text, conv(d.initializer.tokens));
```
Comment (Sobaya007) [inline source/repld/runner.d]: How about the following?
```d
d.initializer ? getType(*decl).format!"(%s).init" : conv(d.initializer.tokens)
```

Comment (kubo39) [reply source/repld/runner.d]: Thanks for advise, fixed.

**Comment by Sobaya007** [comment]
closes #7

**Pull request merged**